service/src/main/java/uk/gov/hmcts/reform/fpl/service/ValidateSupportingEvidenceBundleService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.validation.groups.DateOfIssueGroup;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ValidateSupportingEvidenceBundleService {
+    private final ValidateGroupService validatorService;
+
+    public <T> List<String> validateBundle(List<T> documentBundle) {
```

Comment:
Does it need to be more complex by introducing the generics? 

The code seems to work with List<SupportingEvidenceBundle>

Reply:
@Jamesnelson117 is going to use this for all other supporting documents.

Reply:
Ok, probably he should refactor then as we don't know the final design, but it's fine. Happy to leave it here!

Reply:
@akomisarek @KiranVarma02 no need for generics here. List is always going to be `List<Element<SupportingEvidenceDocument>>`

Reply:
Good :)

Reply:
ok, removing Generic.